js/views/inbox_view.js

```diff
@@ -21,61 +21,16 @@
   Whisper.ConversationStack = Whisper.View.extend({
     className: 'conversation-stack',
     open(conversation) {
-      const id = `conversation-${conversation.cid}`;
-      const container = $('#main-view .conversation-stack');
-
-      // Has been opened since app start, but not focussed
-      const conversationExists = container.children(`#${id}`).length > 0;
-      // Is focussed
-      const conversationOpened = container.children().first().id === id;
-
-      // To limit the size of the DOM for speed
-      const maxNumConversations = 10;
-      const numConversations = container
-        .children()
-        .not('.conversation.placeholder').length;
-      const shouldTrimConversations = numConversations > maxNumConversations;
-
-      if (shouldTrimConversations) {
-        // Removes conversation which has been hidden the longest
-        container.children()[numConversations - 1].remove();
-      }
-
-      if (conversationExists) {
-        // User opened conversation, move it to top of stack, rather than re-rendering
-        const conversations = container
-          .children()
-          .not('.conversation.placeholder');
-        container
-          .children(`#${id}`)
-          .first()
-          .insertBefore(conversations.first());
-        conversation.trigger('opened');
-
-        return;
-      }
+      // const container = $('#main-view .conversation-stack');
+      // container.html('');
 
-      if (!conversationOpened) {
-        this.$el
-          .first()
-          .find('video, audio')
-          .each(function pauseMedia() {
-            this.pause();
-          });
-        let $el = this.$(`#${id}`);
-        if ($el === null || $el.length === 0) {
-          const view = new Whisper.ConversationView({
-            model: conversation,
-            window: this.model.window,
-          });
-          view.view.resetScrollPosition();
-
-          // eslint-disable-next-line prefer-destructuring
-          $el = view.$el;
-        }
+      this.setupSessionConversation(conversation.id);
```

Comment:
`conversation.id` is passed here but the function doesn't take any arguments. 

Also does it create a store every-time you open a conversation?